emukit/examples/README.md

```diff
@@ -0,0 +1,55 @@
+# Emukit Examples
+
+This page contains a curated list of Emukit examples and links to other tutorials. It is heavily inspired on [contributing
+examples](https://github.com/apache/incubator-mxnet/blob/master/example/README.md) section of [MXnet](https://mxnet.apache.org/).
+ 
+  - [Contributing](#contributing)
+  - [List of examples](#list-of-examples)
+  - [List of tutorials](#list-of-tutorials)
+
+
+## <a name="Contributing"></a>Contributing
+If you want to contribute to this of, please open a new pull request.
+
+
+### Examples
+
+Examples can be either notebooks that tell a story about a problem/question using Emukit, *e.g* the analysis of the properties of a simulator, or
+they contain the implementation of some specific method. Examples can live in a .py file, and ideally have tests and 
+come with an illustrative notebook. 
+
+#### Examples location
+
+Example applications or scripts should be submitted in this `emukit/examples` folder.  Each example must live in a separated 
+folder that can contain some extra files an dependencies. Please make sure that you update this `README.md` file with the information 
+about you example when submitting and PR with an example.
+
+
+#### Examples tests
+
+As part of making sure all our examples are running correctly with the latest version of Emukit, yor can add your own tests 
+here `emukit/tests/examples/test_example.py`. (If you forget, don't worry your PR will not pass the sanity check).
```

Comment:
Not sure about that. If there are no tests, PR will still pass the build alright. But you could say "if you forget, don't worry, we'll remind you during the review"

Reply:
done